src/test/fuzz/tx_pool.cpp

```diff
@@ -186,6 +186,16 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         RegisterSharedValidationInterface(txr);
         const bool bypass_limits = fuzzed_data_provider.ConsumeBool();
         ::fRequireStandard = fuzzed_data_provider.ConsumeBool();
+
+        // Make sure ProcessNewPackage on one transaction works and always fully validates the transaction.
+        // The result is not guaranteed to be the same as what is returned by ATMP.
+        const auto result_package = WITH_LOCK(::cs_main,
+                                    return ProcessNewPackage(node.chainman->ActiveChainstate(), tx_pool, {tx}, true));
```

Comment:
Have you look to actually pass chain of transactions from fuzzed data to `ProcessNewPackage` ? I would be glad to have enhanced fuzz coverage of package logic, at least in the future.

Reply:
I thought that would make more sense for actual package submission. I'm not sure if it's that much of a value-add with test accepts, since the mempool doesn't change.